Marlin/src/lcd/dogm/marlinui_DOGM.cpp

```diff
@@ -461,8 +461,11 @@ void MarlinUI::clear_lcd() { } // Automatically cleared by Picture Loop
 
     void MenuItem_sdbase::draw(const bool sel, const uint8_t row, PGM_P const, CardReader &theCard, const bool isDir) {
       if (mark_as_selected(row, sel)) {
-        if (isDir) lcd_put_wchar(LCD_STR_FOLDER[0]);
-        constexpr uint8_t maxlen = LCD_WIDTH - 1;
```

Comment:
One character position was reserved for directory icon, but for regular files this leads to one place of filename were always blank.